src/include/wiredtiger.in

```diff
@@ -799,6 +799,17 @@ struct __wt_session {
 	 */
 	int __F(reconfigure)(WT_SESSION *session, const char *config);
 
+	/*!
+	 * Initiate a single operation to manage shared storage.
+	 *
+	 * @param session the session handle
+	 * @configstart{WT_SESSION.share_storage, see dist/api_data.py}
+	 * @config{force, force sharing of all data., a boolean flag; default \c false.}
+	 * @configend
+	 * @errors
+	 */
+	int __F(share_storage)(WT_SESSION *session, const char *config);
+
```

Comment:
Can we put this in a
```
#if !defined(DOXYGEN)
```
block?  @keitharnoldsmith and I discussed that this would be a good way to not yet expose these APIs that are still work in progress.